src/extractor/restriction_parser.cpp

```diff
@@ -51,8 +50,9 @@ RestrictionParser::RestrictionParser(ScriptingEnvironment &scripting_environment
  * Tries to parse a relation as a turn restriction. This can fail for a number of
  * reasons. The return type is a boost::optional<T>.
  *
- * Some restrictions can also be ignored: See the ```get_exceptions``` function
- * in the corresponding profile.
+ * Some restrictions can also be ignored: See the ```get_restrictions``` function
```

Comment:
The type is the same, yes. As in `restriction:<type>=` or `except=type`.

Well, `except=` also allows for a list of types, as in `except=motorcar,bus` but we handle this below, too.

http://wiki.openstreetmap.org/wiki/Relation:restriction